src/amo/components/Collection/index.js

```diff
@@ -308,10 +311,16 @@ export const mapStateToProps = (
       hasPermission(state, COLLECTIONS_EDIT);
   }
 
+  let collectionToLoad = loading;
```

Comment:
This is an odd variable name as it makes me think it will contain a collection (i.e., the collection to load).

Reply:
lol..yes i think i need to improve on my var naming skills :).